tests/e2e/orgs_test.go

```diff
@@ -0,0 +1,123 @@
+package e2e_test
```

Comment:
I think it would be clearer to use build tags to exclude these tests, rather than requiring that the caller `grep -v e2e`. With this approach, you can no longer run `go test ./...` without also running the e2e tests. https://mickey.dev/posts/go-build-tags-testing/

Reply:
Yep - good idea.